pkg/agent/nodeportlocal/portcache/port_table_others.go

```diff
@@ -1,5 +1,5 @@
-//go:build linux
-// +build linux
+//go:build !windows
+// +build !windows
```

Comment:
that's fine with me, we use this in several other places in the Antrea codebase IIRC